docs/user/intro/doctools.rst

```diff
@@ -0,0 +1,33 @@
+Supported tools
+===============
+
+Read the Docs provides hosting for static html documentation generated by Sphinx,
+MkDocs, Docusaurus and a variety of other tools.
+
+Here are minimal configuration examples for some common tools,
+with more coming soon.
+
+.. grid:: 2
+
+    .. grid-item-card::  Material for MkDocs
+        :link: /intro/mkdocs.html
```

Comment:
```suggestion
        :link: intro/mkdocs.html
```

These links should be relative

Reply:
It would make sense, but your suggestion doesn't work. I also attempted to use an RST ref/doc but neither of those worked for me,hence the hard coded html :(

Reply:
Never mind :man_facepalming: